backend/app/database/falkor/code-graph-backend/api/analyzers/java/analyzer.py

```
@@ -9,6 +9,7 @@
 
 import tree_sitter_java as tsjava
 from tree_sitter import Language, Node
+from tree_sitter import QueryCursor, Query, Node
```

Ruff (0.14.11)

Redefinition of unused `Node` from line 11: `Node` redefined here

Remove definition: `Node`

(F811)